src/main/java/org/fundacionjala/at15/spaceinvaders/Board.java

```diff
@@ -41,8 +47,8 @@ public void createAliens(int rows, int columns) {
         for (int yIndex = 0; yIndex < rows; yIndex++) {
             for (int xIndex = 0; xIndex < columns; xIndex++) {
                 Alien alien = new Alien(
-                    ALIEN_INIT_X + ALIEN_SEPARATION * xIndex,
-                    ALIEN_INIT_Y + ALIEN_SEPARATION * yIndex, ALIEN_WIDTH, ALIEN_HEIGHT);
+                        ALIEN_INIT_X + ALIEN_SEPARATION * xIndex,
+                        ALIEN_INIT_Y + ALIEN_SEPARATION * yIndex, ALIEN_WIDTH, ALIEN_HEIGHT);
```

Comment:
there are unnecessary spaces

Reply:
They are not spaces, the code editor saves like this when the line is very long.